src/Interpreters/Streaming/Aggregator.h

```diff
@@ -1197,13 +1235,25 @@ class Aggregator final
 public:
     void checkpoint(const AggregatedDataVariants & data_variants, WriteBuffer & wb);
     void recover(AggregatedDataVariants & data_variants, ReadBuffer & rb);
+
+    void doCheckpoint(const AggregatedDataVariants & data_variants, WriteBuffer & wb);
+    void doRecover(AggregatedDataVariants & data_variants, ReadBuffer & rb);
+
+    /// [Legacy]
+    void doCheckpointLegacy(const AggregatedDataVariants & data_variants, WriteBuffer & wb);
+    void doRecoverLegacy(AggregatedDataVariants & data_variants, ReadBuffer & rb);
     void recoverStates(AggregatedDataVariants & data_variants, BlocksList & blocks);
     void recoverStatesWithoutKey(AggregatedDataVariants & data_variants, BlocksList & blocks);
     void recoverStatesSingleLevel(AggregatedDataVariants & data_variants, BlocksList & blocks);
```

Comment:
use 'xxxxStatexxxx' as function name. State is uncountable noun.

Reply:
hmm, this is a common naming in Aggregator.

Reply:
Such methods are added by proton team. please correct them.